From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Zeno Albisser <zeno.albisser@digia.com>
Date: Wed, 3 Jul 2013 11:16:56 +0200
Subject: Do not warn for header-hygiene.

QtDeclarative is consciously leaking the namespace QtQml.
Therefore clang would warn about header hygiene.
Unfortunately this cannot be fixed in QtDeclarative,
unless we accept not to be able to use QtQuick1 and QtQuick2
in the same application.
---
 build/common.gypi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/common.gypi b/build/common.gypi
index 89f86bb..eff4b61 100644
--- a/build/common.gypi
+++ b/build/common.gypi
@@ -4070,6 +4070,7 @@
                 '-Wno-unused-value',
                 '-Wno-unused-variable',
                 '-Wno-unused-function',
+                '-Wno-header-hygiene',
               ],
             }],
             ['clang==1 and clang_use_chrome_plugins==1', {
